app/views/groups/index.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="flex h-full flex-col overflow-y-scroll bg-white shadow-xl  max-w-screen-lg px-4 py-2 mx-auto">
+  <div class="flex-1 overflow-y-auto py-2 px-4 sm:px-6">
+    <div class="flex items-start justify-between">
+      <h2 class="text-lg font-medium text-gray-900" id="slide-over-title">Groups</h2>
+      <%= link_to "New Category", categories_path, class: "inline-flex justify-center border border-transparent cursor-pointer bg-secondColor py-2 px-4 text-sm font-medium text-white shadow-sm focus:outline-none focus:ring-2 focus:ring-secondColor focus:ring-offset-2"%>
+    </div>
```

Comment:
- [ **OPTIONAL** ] According to the design, the project requirement and what I mentioned in the previous comment, I think it would be better if you remove this button as I can't see something like this in both design and project requirement. As mentioned in the requirements Home page should be for **categories** for you, you use it for **Group**. 
**I suggest you remove this button, change the page title, "Group" to "CATEGORIES", then button at the bottom for `creating group`, change its text to `Add New Category` as I also mentioned in the previous comment and style it correctly**